include/gatekeeper_lls.h

```diff
@@ -205,6 +205,12 @@ struct lls_config {
 	/* Log ratelimit burst size for LLS block. */
 	uint32_t          log_ratelimit_burst;
 
+	/* The rate and burst size of the icmp messages. */
+	uint32_t          front_icmp_msgs_per_sec;
+	uint32_t          front_icmp_msgs_burst;
+	uint32_t          back_icmp_msgs_per_sec;
+	uint32_t          back_icmp_msgs_burst;
```

Comment:
These fields are better off in `struct gatekeeper_if`.

Reply:
I just realized that you are following the GK model. Let's keep these fields here to keep LLS and GK in a similar fashion.

Reply:
Sure.